lib/dslash_gamma_helper.cu

```diff
@@ -10,21 +10,24 @@ namespace quda {
     ColorSpinorField &out;
     const ColorSpinorField &in;
     const int d;
+    const int proj;
     unsigned int minThreads() const { return in.VolumeCB(); }
 
   public:
-    GammaApply(ColorSpinorField &out, const ColorSpinorField &in, int d) :
+    GammaApply(ColorSpinorField &out, const ColorSpinorField &in, int d, int proj = 0) :
       TunableKernel2D(in, in.SiteSubset()),
       out(out),
       in(in),
-      d(d)
+      d(d),
+      proj(proj)
     {
       apply(device::get_default_stream());
     }
 
     void apply(const qudaStream_t &stream) {
       TuneParam tp = tuneLaunch(*this, getTuning(), getVerbosity());
-      launch<Gamma>(tp, stream, GammaArg<Float, nColor>(out, in, d));
+      if(proj == 0) launch<Gamma>(tp, stream, GammaArg<Float, nColor>(out, in, d, 0));
```

Comment:
Maybe make the last argument to `GammaArg` default to 0 to avoid needing to specify 0 here and similarly on line 43 below?

Reply:
addressed in 0847503